hooks/useTopics/index.js

```diff
@@ -69,6 +72,7 @@ export const TopicsProvider = ({ children }) => {
         addQuizItems: (value) => dispatch({ type: 'addQuizItems', value }),
         userAnswers: state.userAnswers,
         addUserAnswers: (value) => dispatch({ type: 'addUserAnswers', value }),
+        resetTopics: (value) => dispatch({ type: 'resetTopics' }),
```

Comment:
En este caso no usamos un `value` ya que la acción es hacer un reset. Medio extraño que no te haya dicho nada el `eslint`. 🤔 
```suggestion
        resetTopics: () => dispatch({ type: 'resetTopics' }),
```

Reply:
okas... bueno me base en otro script de favoritos y lo dejé así... ahora lo corrijo :)

Reply:
Tienes razón, se nos paso en Favoritos jaja, 🥲